pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -60,6 +61,9 @@
                                     <%= question.title %></a>
                                 <%- include('../partials/issueBadge', {count: question.open_issue_count, issueQid: question.qid}); %>
                             </td>
+                            <td class="align-middle">
+                                <image src="<%= urlPrefix %>/course_admin/questionThumbnail/<%= question.qid %>" onerror="this.style.display='none'" width="100" height="100"/>
```

Comment:
What happens when the browser renders? Does it initially render with the size set to `100 x 100`, then if it turns out there is no thumbnail, does it hide it and cause the table to resize? If that is what happens then it might be better to not request thumbnails for questions that don't have them.

Reply:
That is how it works currently. In order to not request the thumbnail, would that require adding something like a thumbnail tag to the info.json? So that we can check if the question has a thumbnail without having to load in the link. Alternatively, is there a way that it is initially no size, but then resizes to 100x100 upon the image loading?

Reply:
I think we want to avoid adding thumbnail information to `info.json`, because that would be annoying for instructors. We could either: (1) check for thumbnail presence when rendering the page and embed the exact filename in the HTML, or (2) check for thumbnail presence when syncing the course to the DB and store the filename in the `questions` table.

I would suggest trying (1) and measuring how fast it is on a questions list with many questions. If that's too slow then we might have to resort to (2), which is more complicated.

I also suggest that maybe 64x64 might be a good size. The current size seems visually a bit too big to me.